community/libplist/test.patch

```diff
@@ -0,0 +1,9 @@
+--- libplist-2.0.0/test/emptycmp.test.orig	2019-05-11 14:12:09.323356191 +0200
++++ libplist-2.0.0/test/emptycmp.test	2019-05-11 14:12:22.723356503 +0200
+@@ -4,4 +4,6 @@
+ DATAOUT=$top_builddir/test/data
+ TESTFILE=1.plist
+ 
++echo "$(pwd): $top_builddir/test/plist_cmp $DATASRC/$TESTFILE $DATAOUT/$TESTFILE.out"
++
+ $top_builddir/test/plist_cmp $DATASRC/$TESTFILE $DATAOUT/$TESTFILE.out
```

Comment:
I don't understand what the purpose of this patch is.

Reply:
Sorry, needed the output in CI and removed the file from the APKBUILD but forgot delete the actual file.